chroma-manager/tests/utils/install-3.x.exp

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/expect
```

Comment:
do we have a version of this file for 4.x?

Reply:
The version that already exists is the one for 4.x.